CHANGELOG.md

```diff
@@ -1,8 +1,39 @@
+## 0.5.0
+
+### New Additions
+
+* Much improved test helpers ([#109])
+* `Bamboo.TaskSupervisorStrategy` is now started by default ([#133])
+* New Mailgun adapter ([#125])
+* Link to new Sparkpost adapter ([#118])
+* Shared mode for working with multiple process/acceptance tests ([#136])
+* New `Bamboo.Phoenix.put_layout/2` for setting HTML and text layouts at the same time ([#122])
```

Comment:
I thought so too, but I guess they don’t :( I viewed it and there were no links.
> On Apr 29, 2016, at 4:25 PM, Tute Costa <notifications@github.com> wrote:
> 
> In CHANGELOG.md <https://github.com/thoughtbot/bamboo/pull/140#discussion_r61639570>:
> 
> > @@ -1,8 +1,39 @@
> > +## 0.5.0
> > +
> > +### New Additions
> > +
> > +* Much improved test helpers ([#109])
> > +* `Bamboo.TaskSupervisorStrategy` is now started by default ([#133])
> > +* New Mailgun adapter ([#125])
> > +* Link to new Sparkpost adapter ([#118])
> > +* Shared mode for working with multiple process/acceptance tests ([#136])
> > +* New `Bamboo.Phoenix.put_layout/2` for setting HTML and text layouts at the same time ([#122])
> I thought GitHub autolinked these.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/thoughtbot/bamboo/pull/140/files/106b4a7277374f2306b2bb4aaddcd8e876ea2887#r61639570>